documentation/man/features/build.rst

```diff
@@ -83,6 +84,9 @@ OPTIONS
   build process, including configuration files and script output. This option also
   considers whether or not the user is using kw env.
 
+-v, \--verbose:
+  Verbose mode allows the user to see the commands executed under the hood.
```

Comment:
Let's use something more formal in the documentation. Something like this:

Verbose mode. Causes kw to print debugging messages about its progress. This is helpful in debugging issues.

(Same idea for all commits)